TODO.txt

```diff
@@ -107,3 +107,5 @@ Other
   ``skimage.morphology._util._fast_pad``.
 * When sphinx-gallery>=0.9.0, remove the thumbnail_size in
   doc/source/conf.py as the default value will be comparable (#4801).
+* When ``scipy`` is set to >= 1.16, remove legacy (i.e. non-zoom) code paths in
+``skimage.transform.resize``.
```

Comment:
Thanks to NEP-29, we can now do date-based reminders (🎉!): this will be done on Jan 1 2022. Should we add a separate time-based section to the todos?

Reply:
good idea, yes

Reply:
Oops — that should be 2023. Time flies when you're having fun! =P

Reply:
See how this latest commit reorganizing the "Other" section looks. 

We can actually do many of the 2021 items now (we already require NumPy 1.16 and we SciPy 1.1 was released in May 2018).